stdlib/2and3/datetime.pyi

```diff
@@ -1,31 +1,39 @@
 import sys
 from time import struct_time
-from typing import Optional, SupportsAbs, Tuple, overload
+from typing import (
+    AnyStr, Optional, SupportsAbs, Tuple, Union, overload,
+    ClassVar,
+)
 
-MINYEAR = 0
-MAXYEAR = 0
+if sys.version_info >= (3,):
+    _Text = str
+else:
+    _Text = Union[str, unicode]
+
+MINYEAR: int
+MAXYEAR: int
 
 class tzinfo:
     def tzname(self, dt: Optional[datetime]) -> str: ...
     def utcoffset(self, dt: Optional[datetime]) -> Optional[timedelta]: ...
     def dst(self, dt: Optional[datetime]) -> Optional[timedelta]: ...
     def fromutc(self, dt: datetime) -> datetime: ...
 
-class timezone(tzinfo):
-    utc = ...  # type: timezone
-    min = ...  # type: timezone
-    max = ...  # type: timezone
+if sys.version_info >= (3, 2):
+    class timezone(tzinfo):
+        utc: ClassVar['timezone']
```

Comment:
One more minor note: you don't need to quote forward references in stubs. Just write ``ClassVar[timezone]`` etc.